test/Microsoft.AspNetCore.Blazor.Build.Test/RazorCompilerTest.cs

```diff
@@ -525,23 +525,23 @@ public void SupportsLayoutDeclarationsViaTemporarySyntax()
             Assert.NotNull(layoutAttribute);
             Assert.Equal(typeof(TestLayout), layoutAttribute.LayoutType);
             Assert.Collection(frames,
-                frame => AssertFrame.Text(frame, "\nHello"));
+                frame => AssertFrame.Text(frame, "Hello"));
```

Comment:
The extra leading newline looked to me like it's an artifact of your workaround for not having proper directive support. If this isn't the case, let me know and I'll look into it.

Reply:
Exactly. Nobody really wants spurious extra newlines :)